src/analysis/retail/rogue/outlaw/modules/features/Checklist/Component.tsx

```diff
@@ -138,10 +137,10 @@ const OutlawRogueChecklist = ({
           <AbilityRequirement spell={TALENTS.KILLING_SPREE_TALENT.id} />
         )}
         <AbilityRequirement spell={SPELLS.VANISH.id} />
-        {combatant.hasCovenant(COVENANTS.NIGHT_FAE.id) && (
+        {combatant.hasTalent(TALENTS.SEPSIS_TALENT.id) && (
           <AbilityRequirement spell={TALENTS.SEPSIS_TALENT.id} />
         )}
-        {combatant.hasCovenant(COVENANTS.VENTHYR.id) && (
+        {combatant.hasTalent(TALENTS.FLAGELLATION_TALENT.id) && (
```

Comment:
I believe Outlaw rogue no longer has access to Flagellation - it's a Sub spec talent.

Reply:
@ToppleTheNun I will remove it. I thought it was for all specs.